### PR TITLE
Fix stylelint errors

### PR DESCRIPTION
**Pull request by guastallaigor**
**Description**
This PR fix some stylelint errors and adds the command `npm run prebuild:stylelint` to husky pre-commit hook, because a few lint errors is passing some PR's.

**Compatibility**
After every commit, stylelint will be run and fixed.

**Caveats**
I'm not entirely sure if `prebuild:stylelint` is the best option here. Maybe just `stylelint` to show the errors and stop the commit. If you think it's better, feel free to change or ask me to do so :+1:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -107,7 +107,7 @@
   "husky": {
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
-      "pre-commit": "lint-staged"
+      "pre-commit": "npm run prebuild:stylelint && lint-staged"
```
Comment (BcRikko) [inline package.json]: Thanks 😆 

The package.json should look like this:
```json
  "husky": {
    "hooks": {
      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
      "pre-commit": "lint-staged"      // <-- remove `npm run prebuild:stylelint &&`
    }
  },
  "lint-staged": {
    "scss/**/*.scss": [
      "npm run prebuild:stylelint",     // <-- add
      "npm run postbuild",
      "git add"                         // <-- add
    ],
    "*.js": [
      "eslint '.storybook/**/*.js' 'story/**/*.js'"
    ]
  },
```

**Comment by guastallaigor** [comment]
@BcRikko Thank you, sorry about the delay :frowning_face:

**Comment by BcRikko** [comment]
Don't worry about it 👍

**Pull request merged**